### PR TITLE
LPD-36055 Remove FeatureFlag LPD-36053 (Journal Undo/Redo/History buttons)

### PR DESCRIPTION
**Pull request by NemethNorbert**
Description:
This PR is removes the Undo All Button and the Feature Flag LPD-36053 as it is not needed anymore.
It also Rewords LPD-11228 FF description so it is clear both autosave and undo/redo functionality are under that FF.

Test:
Test is not included as not needed, but all the UndoRedo test is updated to not include LPD-36053 anymore

Enable FF  LPD-11228, Undo Redo and the history button should be displayed when editing a web content
Screenshot 2024-10-16 at 15 35 30

Please review the changes

### CODE REVIEW

File: modules/test/playwright/tests/journal-web/journalArticleAutosave.spec.ts [hunk 1]
```diff
@@ -60,7 +60,6 @@ const autoSaveUndoRedoTest = mergeTests(
 	featureFlagsTest({
 		'LPD-11228': true,
 		'LPD-15596': true,
-		'LPD-36053': true,
```
Comment (ambrinchaudhary) [inline modules/test/playwright/tests/journal-web/journalArticleAutosave.spec.ts:63]: Now this is the same than `autoSaveTest`, so I would remove `autoSaveUndoRedoTest` and replace all its uses by  `autoSaveTest`

**Review by ambrinchaudhary — APPROVED** [review]
Just a minor suggestion about tests, the rest LGTM

File: modules/test/playwright/tests/journal-web/journalArticleAutosave.spec.ts (hunk 1 above)
Comment (NemethNorbert) [reply modules/test/playwright/tests/journal-web/journalArticleAutosave.spec.ts:63]: I pushed up another commit to remove autoSaveUndoRedoTest

**Comment by NemethNorbert** [comment]
Hey @ambrinchaudhary,

I resolved the conflict and start the ci:forward process. Thank you for the review

**Comment by NemethNorbert** [comment]
ci:forward

**Review by DiegoHu97 — APPROVED** [review]
LGTM! after solving the SF issue, let's run the test suite to see that everthign is ok

**Comment by NemethNorbert** [comment]
It's quite wierd, the SF is complaining about a file I did not even change. It seems to be caused by this commit:
https://github.com/brianchandotcom/liferay-portal/pull/155186/commits/3cb9c2175140a0968b41ecdb79b3a93e776b94ac

**Comment by NemethNorbert** [comment]
Hey @DiegoHu97 ,

They already merged the SF fix into master, as I wrote above it was caused by that commit. However it needs time to be mirrored.

**Comment by NemethNorbert** [comment]
ci:test:relevant

**Comment by NemethNorbert** [comment]
ci:test:stable

**Comment by NemethNorbert** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:report:44456436

**Comment by DiegoHu97** [comment]
ci:report:44710817

**Comment by DiegoHu97** [comment]
ci:test:relevant

**Comment by DiegoHu97** [comment]
No unique failures, I will forward the pr

**Comment by DiegoHu97** [comment]
ci:forward

**Comment by ambrinchaudhary** [comment]
ci:report:46418330

**Comment by ambrinchaudhary** [comment]
Now there are unique failures to this pull: 
```
com.liferay.dynamic.data.lists.internal.exportimport.test.DDLDisplayExportImportTest,
PortalLogAssertorTest-modules-integration-mysql57/0/4,
com.liferay.object.rest.internal.resource.v1_0.test.OpenAPIResourceTest,
com.liferay.journal.model.listener.test.DDMStructureModelListenerTest,
com.liferay.headless.commerce.delivery.cart.resource.v1_0.test.AttachmentResourceTest,
com.liferay.headless.commerce.delivery.order.resource.v1_0.test.AttachmentResourceTest,
```

But I tend to think that are not related with commits for LPD-36055. I will take the risk and force the forward....

**Comment by ambrinchaudhary** [comment]
ci:forward:force

**Pull request closed without merging**